### DISCUSSION TITLE
How do I exit full screen mode by pressing the Escape key?

### DISCUSSION THREAD

**Discussion by goodwin74** [cefsharp/CefSharp#3361] [Q&A]
I made the video full screen with **IDisplayHandler**. But I can't figure out how to implement the exit from full screen mode when the escape button is pressed?!
I would be grateful for your help!

p.s. Fullscreen implementation was done following the example: [from here](https://github.com/cefsharp/CefSharp/commit/7f1f1c5f214e10c77f3c47740e8ef9ac26830be9#diff-c2d72c89ae2f1668e2302d726a1b58aeR32)

**Comment by amaitland** [comment]
https://www.magpcss.org/ceforum/viewtopic.php?f=6&t=13642 has a discussion on the topic.

Looks like you need to implement http://cefsharp.github.io/api/86.0.x/html/T_CefSharp_IKeyboardHandler.htm and use JavaScript to exit.

https://github.com/cefsharp/CefSharp/wiki/General-Usage#1-how-do-you-call-a-javascript-method-from-net
